### ISSUE TITLE
I Report this Github, App This is Trojan Hacker

### ISSUE THREAD

**Issue by Fandyfr** [maximmax42/Discord-CustomRP#232]
![image](https://github.com/maximmax42/Discord-CustomRP/assets/135007919/047b44ae-f51b-4387-8d87-80b16594215f)

**Comment by maximmax42** [comment]
The app does not have any viruses.
https://docs.customrp.xyz/faq#is-this-a-virus-my-antivirus-virustotal-etc-says-theres-a-virus.

This happened with Defender before, multiple times, but with a different trojan name. Every time I send the file to Microsoft they rescan it and remove the detection flag.

**Comment by Fandyfr** [comment]
Fortunately, when I downloaded it, it was blocked by Ublock, but I didn't care, but I still installed it, as soon as my laptop blinked, a warning appeared, it was a Trojan virus, a hacker attack from the app.

And You Developer is Lie

**Comment by Fandyfr** [comment]
![image](https://github.com/maximmax42/Discord-CustomRP/assets/135007919/8d75dcf8-c554-4c04-837c-a8dcb47bd45a)
See???
![image](https://github.com/maximmax42/Discord-CustomRP/assets/135007919/0529b9df-4339-490a-8d32-21133c16ab38)

**Comment by maximmax42** [comment]
66 out of 70 antiviruses say the file is safe, 4 say it's a virus. That's the definition of "false positive".
Check the code, it's open source for a reason. As for why some antiviruses think there's a virus, it's simple - the app is not widely known and it's not digitally signed (I'm Russian, I can't buy a certificate because of sanctions).

**Comment by Fandyfr** [comment]
your website customrp xyz i install on this website click on installer, his a popup is virus on ublock extension and McAfee

**Comment by maximmax42** [comment]
Yes, there is a popup on the site. Yes, sometimes it has some weird ads. As long as you don't download anything from the popup, you're fine.

**Comment by Fandyfr** [comment]
why do you give dangerous advertisements to other people? that's very illegal

**Comment by maximmax42** [comment]
Ads on the site are managed by an ad network called Adsterra, not by me personally. I'm pretty sure they don't advertise dangerous stuff. There can be people who are abusing the network, but that happens with Google ad network as well.

**Comment by maximmax42** [comment]
> google ad is Safe

Not really. A lot of Indian scammers are using Google Ads to advertise their scam websites and steal millions from people across the world.
Of course, these ads don't stay for long, Google doesn't endorse scams, but the fact still stands, even Google Ads can be abused.

**Comment by Fandyfr** [comment]
why not just use ad, let the app be open source and free to use

**Comment by maximmax42** [comment]
???
The app IS open source and free to use. What are you talking about?

**Comment by PreciselyPixels** [comment]
> why not just use ad, let the app be open source and free to use

The adverts are to support the developer. The app itself is both open source and free to use; hence why it is on github in the first place. Furthermore, you're not required to download the application from the website. A way to download the application is also on the github. Follow the instructions on the readme.

**Comment by ItsbaileyX3525** [comment]
This has to be the silliest conversation I have read lol
